docker/Dockerfile.rootless.linux.amd64

```diff
@@ -1,5 +1,6 @@
-FROM redhat/ubi8-minimal:8.9-1161
-RUN microdnf update && microdnf install ca-certificates git tar openssh curl perl python38 shadow-utils
+FROM us.gcr.io/platform-205701/harness/ubi8:8.8
+USER root
```

Comment:
can you test in rootless mode and make sure this is working fine ?

Reply:
may be we need to switch to non root user after installing packages ?

Reply:
Yes @Sapa96  Tested in rootless image in devspace.